lib/auth/Vault.ts

```diff
@@ -18,7 +18,10 @@ function vaultSignatureCb(
             body: {
                 userInfo: AuthInfoType,
                 authorizationResults: { [key: string]: any },
-                accountQuota: number,
+                accountQuota: {
```

Comment:
API format changed?
this is not just passing a string instead of number (or ideally `number|string` for backwards compatibility), but a real compatibility break...

Reply:
No, the type was wrong from the beginning, it was already an object with number, now it's a string.

Reply:
It would have created errors if we had TS in cloudserver.